### ISSUE TITLE
[Bug] grass.jupyter InteractiveMap: wrong reprojection for latlong maps

### ISSUE THREAD

**Issue by veroandreo** [OSGeo/grass#3104]
**Describe the bug**
A latlong map displayed with grass.jupyter InteractiveMap class looks shifted by some kilometers. Displaying the same map in QGIS over OpenStreetMap basemap looks just fine. I assume something goes wrong in the reprojection step for 4326 locations. I haven't observed this issue with projected locations. 

**Screenshots**

Within a notebook with grass.jupyter InteractiveMap class:
![image](https://github.com/OSGeo/grass/assets/20075188/68ee9b31-4e9f-4a98-91e8-4b158f8207f0)

The same map in QGIS (opened via GRASS provider):
![image](https://github.com/OSGeo/grass/assets/20075188/06d8d409-1ad3-44c7-a148-79f69286d3b7)

**System description:**
GRASS version: 8.2.1                                                            
Code revision: exported                                                         
Build date: 2023-01-21                                                          
Build platform: x86_64-redhat-linux-gnu                                         
GDAL: 3.5.2                                                                     
PROJ: 9.0.1                                                                     
GEOS: 3.11.0                                                                    
SQLite: 3.40.0                                                                  
Python: 3.11.4                                                                  
wxPython: 4.2.0                                                                 
Platform: Linux-6.3.12-100.fc37.x86_64-x86_64-with-glibc2.36

**Comment by veroandreo** [comment]
This is weird. I used t.rast.series to aggregate the whole strds and display the result, and this one looks good:
![image](https://github.com/OSGeo/grass/assets/20075188/641ae231-034d-4762-a374-9b39b16e0519)

Any single map I tested however, looks shifted as above... :exploding_head:

**Comment by petrasovaa** [comment]
Well, have you tried rendering these 2 rasters with gj.Map or in GUI? I think we need more info, this looks difficult to reproduce.

**Comment by veroandreo** [comment]
> Well, have you tried rendering these 2 rasters with gj.Map or in GUI? I think we need more info, this looks difficult to reproduce.

Yes, in the GUI with a basemap from d.wms they look just fine, they both overlap properly. And they also look good with gj.Map class in the notebook. It's a real mistery...

**Comment by petrasovaa** [comment]
Could you share the raster, that would be probably easiest way to test this.

**Comment by veroandreo** [comment]
Yes, sure! Here are the exported tif files: 

chirps_mean: https://mega.nz/file/4E0AxZIB#SWBa6l7NAIIS3cOadvUrh1Zffv0UsBHW_3TTDU6JaQg
chirps_v2_0_2010_09: https://mega.nz/file/YMNlmAgK#H3F_VVBmfqkeof1WvIp04SJZWf00ioEOPRHtQ3Ag8gY

To be imported in a EPSG:4326 location.

**Comment by petrasovaa** [comment]
> chirps_v2_0_2010_09: https://mega.nz/file/YMNlmAgK#H3F_VVBmfqkeof1WvIp04SJZWf00ioEOPRHtQ3Ag8gY

I tried this one and it looks fine for me. Perhaps you could investigate more on your side, check r.info, try use_region=False, not sure what else...

**Comment by petrasovaa** [comment]
I will close this as not reproducible, but please feel free to reopen.
